tests/integration/src/test/java/org/apache/pulsar/tests/integration/containers/DebeziumMongoDbContainer.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.tests.integration.containers;
+
+import org.testcontainers.containers.wait.strategy.HostPortWaitStrategy;
+
+public class DebeziumMongoDbContainer extends ChaosContainer<DebeziumMongoDbContainer> {
+
+    public static final String NAME = "debezium-mongodb-example";
+
+    public static final Integer[] PORTS = { 27017 };
+    private static final String IMAGE_NAME = "debezium/example-mongodb:latest";
```

Comment:
Does image consider using a fixed version e.g debezium/example-mongodb:0.10? Because the latest version is always likely to change.

Reply:
I’m so sorry to trouble you，I should test before pull request. But when i run test in my local environment which is win+docker ，it's not worker.

Reply:
I haven't tested it on windows yet. If it's on mac or linux, use the following steps:

```
mvn install -DskipTests -Pdocker
mvn test -pl  tests/integration -DintegrationTests -DredirectTestOutputToFile=false -Dtest=org.apache.pulsar.tests.integration.functions.PulsarFunctionsProcessTest
```

You can't work locally, you can change it according to my comment, I think it can pass the integration test.